src/main/java/PersonalBudget/business/expense/domain/mapper/ExpenseMapper.java

```diff
@@ -13,7 +13,7 @@ public ExpenseEntity mapExpenseDTOToExpenseEntity(ExpenseDTO expenseDTO, Long us
                 .expenseCategoryId(expenseDTO.expenseCategoryId())
                 .expensePaymentMethodId(expenseDTO.paymentMethodId())
                 .amount(expenseDTO.amount())
-                .expenseDate(expenseDTO.expenseDate())
+                .date(expenseDTO.date())
```

Comment:
data is a very vague name, expenseDate was more indicative. Why did you change it? If you want to have something simpler yet clear I suggest `timestamp`